axoupdater/src/lib.rs

```diff
@@ -211,7 +211,7 @@ impl AxoUpdater {
 
         // Looks like this EXE comes from a different source than the install
         // receipt
-        if current_exe_root != self.install_prefix_root()? {
+        if current_exe_root != self.install_prefix_root_normalized()? {
```

Comment:
You could also consider using `same_file` for this.

Reply:
filed https://github.com/axodotdev/axoupdater/issues/51, thanks for the tip!

Reply:
No prob, we use it successfully in uv for a similar kind of check.